w3sper.js/tests/contract_call_test.js

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+import {seeder, test, Treasury} from "./harness.js";
+import {AccountSyncer, Bookkeeper, Network, ProfileGenerator} from "@dusk/w3sper";
+
+test("account contract call transfer", async () => {
```

Comment:
This test doesn't have any assertion: it just logs things in the console.

Reply:
done, added assertions